src/uwtools/tests/config/test_core.py

```diff
@@ -862,14 +862,16 @@ def test_Config_characterize_values(nml_cfgobj):
     assert template == ["    p3: {{ n }}"]
 
 
-def test_Config_str_to_type(nml_cfgobj):
-    for x in ["true", "yes", "yeah"]:
-        assert nml_cfgobj.str_to_type(x) is True
-    for x in ["false", "no", "nope"]:
-        assert nml_cfgobj.str_to_type(x) is False
-    assert nml_cfgobj.str_to_type("88") == 88
-    assert nml_cfgobj.str_to_type("3.14") == 3.14
-    assert nml_cfgobj.str_to_type("NA") == "NA"  # no conversion
+def test_Config_reify_scalar_str(nml_cfgobj):
+    for x in ["true", "yes", "TRUE"]:
+        assert nml_cfgobj.reify_scalar_str(x) is True
+    for x in ["false", "no", "FALSE"]:
+        assert nml_cfgobj.reify_scalar_str(x) is False
+    assert nml_cfgobj.reify_scalar_str("88") == 88
+    assert nml_cfgobj.reify_scalar_str("3.14") == 3.14
+    assert nml_cfgobj.reify_scalar_str("NA") == "NA"  # no conversion
+    assert nml_cfgobj.reify_scalar_str("cycle: '06'") == "cycle: '06'"
```

Comment:
I know this was the example I gave, but I feel like it's kind of obvious that this whole string would remain a string while having just
```
assert nml_cfgobj.reify_scalar_str("'06'") == "'06'"
```
or even just
```
assert nml_cfgobj.reify_scalar_str("'88'") == "'88'"  # quoted int not converted
```
placed next to the other `88` test might make the intent clearer.

Reply:
Thats fair - I was using the full entry as the example for its own context, but no need to use the tests to prove a point